### DISCUSSION TITLE
应用商店nacos集群配置

### DISCUSSION THREAD

**Discussion by ayebuemo** [1Panel-dev/1Panel#6472] [General]
我发现在面板应用商店,下载的naocs应用
找不到它的配置文件,逐级搜索了也没找到
nacos\conf目录下的 
application.properties 文件  去开启集群数据源设置
也没有cluster.conf 文件 去配置集群节点列表

**Comment by wanghe-fit2cloud** [comment]
进入应用安装目录中查看，是否有持久化出来：
![image](https://github.com/user-attachments/assets/035cfda1-f475-4f35-9c19-428627bcf777)

**Reply by ayebuemo** [reply]
![nacos](https://github.com/user-attachments/assets/b2efc0b4-6375-465e-a82d-f8b50e5a0981)
正常来说nacos下边应该有conf文件夹
里面有配置文件
但这个连这个目录也没有

**Reply by wanghe-fit2cloud** [reply]
![image](https://github.com/user-attachments/assets/3185a341-1605-49d2-9bdb-f4940378db22)
